requirements-dev.txt

```diff
@@ -6,6 +6,7 @@ networkx>=3.1
 pandas>=1.5.2
 plotly>=5.11.0
 pydot>=1.3.0  # required by third_party/param/
+pytest>=7.4.4
```

Comment:
@briancoutinho we use unittest for our testing framework. To keep the maintanence overhead low I suggest not adding another testing framework. 
https://github.com/facebookresearch/HolisticTraceAnalysis/blob/main/.circleci/config.yml#L78-L84